### PR TITLE
[IMP] mail: allow shift+tab to select previous chat window

### PR DESCRIPTION
**Pull request by seb-odoo**
task-2171879

### CODE REVIEW

File: addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js [hunk 1]
```diff
@@ -190,6 +234,16 @@ class ChatWindowManager extends Component {
...
     //--------------------------------------------------------------------------
 
+    /**
+     * Called when a chat window asks to focus the previous chat window.
+     *
+     * @private
+     * @param {CustomEvent} ev
+     * @param {string} ev.detail.currentChatWindowLocalId
+     */
+    _onFocusPreviousChatWindow(ev) {
```
Comment (alexkuhn) [inline addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js]: `_onFocusPrevious*` after `_onFocusNext*`

File: addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js [hunk 2]
```diff
@@ -119,6 +119,50 @@ class ChatWindowManager extends Component {
...
+     * Cycles to the next possible chat window starting from the
+     * `currentChatWindowLocalId`, following the natural order based on the
+     * current text direction, and with the possibility to `reverse` based on
+     * the given parameter.
+     *
+     * @private
+     * @param {integer} currentChatWindowLocalId
+     * @param {boolean} reverse
+     */
+    _cycleChatWindow(currentChatWindowLocalId, reverse = false) {
```
Comment (alexkuhn) [inline addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js]: prefer object for options, e.g. `{ reverse = false } = {}`

**Comment by alexkuhn** [comment]
also missing task-id on commit message ;)

File: addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js (hunk 1 above)
Comment (seb-odoo) [reply addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js]: ok

File: addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js (hunk 2 above)
Comment (seb-odoo) [reply addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js]: ok

**Comment by seb-odoo** [comment]
all done here

**Review by alexkuhn — APPROVED** [review]

**Pull request merged**